### PR TITLE
Expose the generated breaking changes as step#outputs

### PR DESCRIPTION
**Pull request by jmatsu**
Thank you for providing this amazing action. 

This pr exposes the generated file annotation entities as the steps's outputs because we'd like to allow following steps to use these entities. It looks redundant to re-run the checker and your file annotation's information is very useful.

### CODE REVIEW

**Comment by jef** [comment]
ha, didn't see this until now 🤦 was needing the same thing 😅 https://github.com/bufbuild/buf-breaking-action/pull/22

**Review by amckinney — APPROVED** [review]
Nice, thanks for the contribution! I'll go ahead and stamp this one as-is and we'll follow-up with some of the documentation changes proposed in #22 in a follow-up. We'll also need to build the `dist/...` files from your changes in `src/...`, but we'll handle that on our end before we cut the release.

I'll wait for one more pair of eyes, but we'll merge this when we're good to go with it! Thanks again!

**Comment by jmatsu** [comment]
Thanks for the quick review! By the way, I never mind if you close this pr because #22 actually comprises this change and the name of the outputs is not a big deal.  😄

**Review by doriable — APPROVED** [review]
I'm on board with this change, thanks for sending this in! :)

**Comment by amckinney** [comment]
> Thanks for the quick review! By the way, I never mind if you close this pr because https://github.com/bufbuild/buf-breaking-action/pull/22 actually comprises this change and the name of the outputs is not a big deal. 😄

No worries, we'll probably need to adjust some of the minor details there, so it's simpler for us to land this one as-is and pull in the changes we need from #22. Thanks again, and keep your eye out for the next release!

**Comment by cyinma** [comment]
hello @jmatsu @jef , these changes is now released! please have a try and let us know if there is any issue, thanks!

**Comment by jef** [comment]
all looks good from my end! thanks for adding to `v1` 😌

**Pull request merged**
